src/main.py

```
@@ -28,7 +30,11 @@ def main():
         ["Username", "Kills", "Deaths", "Assists", "Damage", "ADR", "KAST", "Impact", "Rating"],
         [str(player) for player in players]  
     )
+
+
+
     
+    teams = parser.parse_teams(players)
```

Flake8 (7.2.0)

[error] local variable 'teams' is assigned to but never used

(F841)